.gitignore

```diff
@@ -0,0 +1,53 @@
+HELP.md
+.gradle
+build/
+local-dev/terra-helm/
+local-dev/terra-helmfile/
+!gradle/wrapper/gradle-wrapper.jar
+!**/src/main/**
+!**/src/test/**
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+out/
+
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+
+### VS Code ###
+.vscode/
+
+### Emacs backup files
+*.*~
+
+### Mac directory metadata
+.DS_Store
+
+### OpenAPI CodeGen detritus
+.openapi-generator/
+pom.xml
+
+# Package Files #
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
```

Comment:
Please also add a LICENSE to the root directory. Probably BSD-3 clause (probably copy another, e.g. Workbench if you like.)

Reply:
That's what I would put. Maybe give a heads up to Shimon to double-check.

Reply:
Thanks, verified with Shimon.